estate/models/real_estate.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
```

Comment:
Keep your imports on alphabetical order. It may seem like a detail but it prevent duplicates on larger projects.
```suggestion
from odoo import fields, models
```